bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -509,11 +512,11 @@ class ReplicaImp : public InternalReplicaApi, public ReplicaForStateTransfer {
                                                   PrePrepareMsg& prePrepareMsg,
                                                   uint32_t maxStorageForRequests);
 
-  PrePrepareMsg* createPrePrepareMessage();
+  std::unique_ptr<PrePrepareMsg> createPrePrepareMessage();
```

Comment:
`using PrePrepareMsgUPtr = std::unique_ptr<PrePrepareMsg>`
for all places which wrap with unique ptr.